src/w3c/headers.js

```diff
@@ -385,7 +413,7 @@ export function run(conf) {
       !conf.isNoTrack &&
       !conf.isSubmission
     ) {
-      const msg = "Document on track but no previous version.";
```

Comment:
This one is probably not desired. All other `msg` end with full-stop.

Reply:
This appears to be a bug in core UI:

https://github.com/w3c/respec/blob/49ddeac2e88a635032d65f21a7244350512c1c14/src/core/ui.js#L278

It adds a "." to the hint. We probably don't want that regardless in i18n'zed contexts. I'll remove it.

Reply:
Just in case, fix is now here: https://github.com/w3c/respec/pull/3310/files#diff-b1ee7df5d4991f7198879953fab78023c1ed2dcb5e6d45b2f8994e50d3459c24